File: plugins/devkit/devkit-java-tests/testData/inspections/useCoupleFix/UseCoupleOfFactoryMethodInConstantWhenPairCreateUsed.java
import com.intellij.openapi.util.Pair;

class UseCoupleOfFactoryMethodInConstantWhenPairCreateUsed {
  private static final <warning descr="Replace with 'Couple<String>'">Pair<String, String></warning> ANY = <warning descr="Replace with 'Couple.of()'">Pair.cr<caret>eate("a", "b")</warning>;
}

File: plugins/devkit/devkit-kotlin-tests/testData/inspections/useCoupleFix/UseCoupleOfFactoryMethodInConstantWhenPairCreateUsed.kt
import com.intellij.openapi.util.Pair

object UseCoupleOfFactoryMethodInConstantWhenPairCreateUsed {
  private val ANY: <warning descr="Replace with 'Couple<String>'">Pair<String, String></warning> = <warning descr="Replace with 'Couple.of()'">Pair.cr<caret>eate("a", "b")</warning>
}
